tests/integration.rs

```diff
@@ -223,6 +223,26 @@ fn prepare_vhost_user_net_daemon(
 
     (command, vunet_socket_path)
 }
+fn prepare_swtpm_daemon(tmp_dir: &TempDir) -> (std::process::Command, String) {
+    let swtpm_tpm_dir = String::from(tmp_dir.as_path().join("swtpm").to_str().unwrap());
+    let swtpm_socket_path = swtpm_tpm_dir.clone() + "/swtpm.sock";
+    std::fs::create_dir(&swtpm_tpm_dir).unwrap();
```

Comment:
You can keep everything as a `PathBuf` and convert to string only when needed.

Reply:
done